content/core-features/presence-and-occupancy/topic-presence_ia-update.textile

```diff
@@ -0,0 +1,737 @@
+---
+title: Presence
+meta_description: "Presence events provide clients with information about the status of other clients 'present' on a channel"
+meta_keywords: "Ably, presence, presence events, channel members, presence history."
+<!-- section: realtime -->
+index: 40
+languages:
+  - javascript
+  - nodejs
+  - php
+  - python
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - go
+jump_to:
+  Help with:
+    - #What is presence?what-is-presence
+    - Getting started#getting-started 
+    - Subscribing to states and events#presence-subscribing-states
+    - Presence history#presence-history
+    - Multiple client IDs#presence-multiple-client-id
+    - Handling failures#handling-failures
+    - Presence with unstable connections#unstable-connections
+    - API Reference#api-reference
+---
+
+h2(#what-is-presence). What is presence?
+
+Presence enables clients to be aware of other clients that are currently "present" on a channel. Each member present on a channel has a unique self-assigned client identifier and system-assigned connection identifier, along with an optional payload that can be used to describe the member's status or attributes. Presence allows you to quickly build apps such as chat rooms and multiplayer games by automatically keeping track of who is present in real time across any device.
+
+h2(#getting-started). Getting started
```

Comment:
I think it is unnecessary to have this `h2`, especially as there's no text in this section. You could simply have both "getting started realtime" and "getting started rest" as `h2` and they would appear nicely at the same level in the on-page TOC. There is not much to be gained from the extra heading level. Note, if you do this, subsequent heading levels would need to be changed.